dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -2082,6 +2086,39 @@ public void deleteWorkflow(@ApiParam(hidden = true) @Parameter(hidden = true, na
         }
     }
 
+    @GET
+    @Path("/github/infer/{owner}/{repo}/{ref}")
+    @Timed
+    @UnitOfWork
+    @Operation(description = "Infer the entries in a repository on GitHub.", security = @SecurityRequirement(name = JWT_SECURITY_DEFINITION_NAME))
```

Comment:
Infers the entries **on a reference** in a repository. Seems like a nit, and it is, but it did confuse me when reviewing, as I thought at first you were downloading the entire repo (all branches), which was starting to raise some concerns in my mind about memory utilization.